### PR TITLE
BStats metrics integration

### PR DESCRIPTION
**Pull request by TobiasGrether**
This pull request implements BStats metrics collection.

BStats is an anonymous open-source metrics collection application, which is pretty popular in the Java Edition scene.

Goals of using it is to assess the range of people using WaterdogPE as well as the amount of people using it, their system architectures etc. This can give us important clues in case we ever want to drop support for certain architectures or java versions.

Metrics can be disabled in the server configuration file, however generally people should keep them enabled since they are useful for us.

### CODE REVIEW

File: src/main/java/dev/waterdog/waterdogpe/WaterdogPE.java [hunk 1]
```diff
@@ -43,6 +43,7 @@ public static void main(String[] args) {
         logger.info("§bStarting WaterDogPE proxy software!");
         logger.info("§3Software Version: " + versionInfo.baseVersion());
         logger.info("§3Build Version: " + versionInfo.buildVersion());
+        logger.info("§3Metrics publisher ID: " + versionInfo.metricsId());
```
Comment (Alemiz112) [inline src/main/java/dev/waterdog/waterdogpe/WaterdogPE.java:46]: I don't think this is ever going to change, so we don't need to expose it in a log.

**Review by Alemiz112 — CHANGES_REQUESTED** [review]

**Comment by Alemiz112** [comment]
Would be also nice if we could do players per version pie chart graph.

**Comment by TobiasGrether** [comment]
> Would be also nice if we could do players per version pie chart graph.

Hmm yeah I don't know if i want to mess with bstats more than I absolutely have to. I can maybe integrate that in a future update

**Comment by Alemiz112** [comment]
> Hmm yeah I don't know if i want to mess with bstats more than I absolutely have to. I can maybe integrate that in a future update

Lmao, okay.

**Review by Alemiz112 — APPROVED** [review]

**Pull request merged**